grype/presenter/table/presenter.go

```diff
@@ -81,15 +116,16 @@ func (p *Presenter) Present(output io.Writer) error {
 
 	if p.withColor {
 		for _, row := range rs.Deduplicate() {
-			severityColor := getSeverityColor(row.Severity)
 			table.Rich(row.Columns(), []tablewriter.Colors{
-				{},              // name
-				{},              // version
-				{},              // fix
-				{},              // package type
-				{},              // vulnerability ID
-				severityColor,   // severity
-				annotationColor, // annotations
+				{}, // name
```

Comment:
Do we need these placeholder empty structs now since we don't use the colors? I don't see where else in the PR we write/read from this list. What happens if we just make this `table.Rich(row.Columns)`

Can we simplify this to just `table.Append`?